### DISCUSSION TITLE
LSF: how to get monitor data?

### DISCUSSION THREAD

**Discussion by kruglov-dmitrii** [ansys/pyaedt#3826] [Q&A] [answered]
Hi! 

I am using PyAEDT on Linux, and I submit simulations to LSF cluster. I can successfully submit the simulation using e.g. Hfss.submit_job(), and pyaedt then gives me a job ID. How do I then use this job ID to monitor the simulation progress? Hfss.get_monitor_data() doesn't produce any output, and I cannot find details on it in the documentation.

![LSF_submit_job](https://github.com/ansys/pyaedt/assets/136462756/57f26c4b-67a1-4670-ad0e-cf6f496e1466)

Thanks!

BR,
Dmitrii

**Comment by Samuelopez-ansys** [comment, marked as answer]
@kruglovIFX This is related to: https://github.com/ansys/pyaedt/issues/3855

get_monitor_data() only works for local analysis (not submitted).
